pkg/bigquery/db.go

```diff
@@ -139,6 +139,45 @@ func (d *Client) Select(ctx context.Context, query *query.Query) ([][]interface{
 	return result, nil
 }
 
+func (d *Client) SelectWithSchema(ctx context.Context, query *query.Query) ([]string, [][]interface{}, error) {
```

Comment:
let's make the response to be a struct like:
```go
type QueryResult struct {
    Columns []string
    Rows []interface{}
}
```

that way the function becomes more self-explanatory, and gives us the ability to expand response structures, e.g. adding column types in the future, without having to rewrite all the function/interface signatures.